{"uid":"a05a62543e48e6af","name":"Login with valid credentials test","fullName":"tests.login_test.TestLogin#test_valid_login","historyId":"18657c207d70aa57845e0b7f4751c290","time":{"start":1703639580776,"stop":1703639590971,"duration":10195},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557eec01ad33 <unknown>\n#1 0x557eebcd7f87 <unknown>\n#2 0x557eebd27c0e <unknown>\n#3 0x557eebd27d11 <unknown>\n#4 0x557eebd73004 <unknown>\n#5 0x557eebd5136d <unknown>\n#6 0x557eebd70297 <unknown>\n#7 0x557eebd510e3 <unknown>\n#8 0x557eebd19044 <unknown>\n#9 0x557eebd1a44e <unknown>\n#10 0x557eebfdf861 <unknown>\n#11 0x557eebfe3785 <unknown>\n#12 0x557eebfcd285 <unknown>\n#13 0x557eebfe441f <unknown>\n#14 0x557eebfb120f <unknown>\n#15 0x557eec008028 <unknown>\n#16 0x557eec0081f7 <unknown>\n#17 0x557eec019ed4 <unknown>\n#18 0x7f1672294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7effc4ca4c10>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.flaky(reruns=1)\n    def test_valid_login(self, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"fa5cd6582b6c996811a27061717c780b\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7effc4d60b80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x557eec01ad33 <unknown>\nE       #1 0x557eebcd7f87 <unknown>\nE       #2 0x557eebd27c0e <unknown>\nE       #3 0x557eebd27d11 <unknown>\nE       #4 0x557eebd73004 <unknown>\nE       #5 0x557eebd5136d <unknown>\nE       #6 0x557eebd70297 <unknown>\nE       #7 0x557eebd510e3 <unknown>\nE       #8 0x557eebd19044 <unknown>\nE       #9 0x557eebd1a44e <unknown>\nE       #10 0x557eebfdf861 <unknown>\nE       #11 0x557eebfe3785 <unknown>\nE       #12 0x557eebfcd285 <unknown>\nE       #13 0x557eebfe441f <unknown>\nE       #14 0x557eebfb120f <unknown>\nE       #15 0x557eec008028 <unknown>\nE       #16 0x557eec0081f7 <unknown>\nE       #17 0x557eec019ed4 <unknown>\nE       #18 0x7f1672294ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1703639396359,"stop":1703639396359,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1703639396360,"stop":1703639396360,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"_verify_url","time":{"start":1703639396360,"stop":1703639396360,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557eec01ad33 <unknown>\n#1 0x557eebcd7f87 <unknown>\n#2 0x557eebd27c0e <unknown>\n#3 0x557eebd27d11 <unknown>\n#4 0x557eebd73004 <unknown>\n#5 0x557eebd5136d <unknown>\n#6 0x557eebd70297 <unknown>\n#7 0x557eebd510e3 <unknown>\n#8 0x557eebd19044 <unknown>\n#9 0x557eebd1a44e <unknown>\n#10 0x557eebfdf861 <unknown>\n#11 0x557eebfe3785 <unknown>\n#12 0x557eebfcd285 <unknown>\n#13 0x557eebfe441f <unknown>\n#14 0x557eebfb120f <unknown>\n#15 0x557eec008028 <unknown>\n#16 0x557eec0081f7 <unknown>\n#17 0x557eec019ed4 <unknown>\n#18 0x7f1672294ac3 <unknown>","statusTrace":"self = <tests.login_test.TestLogin object at 0x7effc4ca4c10>\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.flaky(reruns=1)\n    def test_valid_login(self, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/login_test.py:42: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"fa5cd6582b6c996811a27061717c780b\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7effc4d60b80>\nmessage = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x557eec01ad33 <unknown>\nE       #1 0x557eebcd7f87 <unknown>\nE       #2 0x557eebd27c0e <unknown>\nE       #3 0x557eebd27d11 <unknown>\nE       #4 0x557eebd73004 <unknown>\nE       #5 0x557eebd5136d <unknown>\nE       #6 0x557eebd70297 <unknown>\nE       #7 0x557eebd510e3 <unknown>\nE       #8 0x557eebd19044 <unknown>\nE       #9 0x557eebd1a44e <unknown>\nE       #10 0x557eebfdf861 <unknown>\nE       #11 0x557eebfe3785 <unknown>\nE       #12 0x557eebfcd285 <unknown>\nE       #13 0x557eebfe441f <unknown>\nE       #14 0x557eebfb120f <unknown>\nE       #15 0x557eec008028 <unknown>\nE       #16 0x557eec0081f7 <unknown>\nE       #17 0x557eec019ed4 <unknown>\nE       #18 0x7f1672294ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1703639580776,"stop":1703639590971,"duration":10195},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557eec01ad33 <unknown>\n#1 0x557eebcd7f87 <unknown>\n#2 0x557eebd27c0e <unknown>\n#3 0x557eebd27d11 <unknown>\n#4 0x557eebd73004 <unknown>\n#5 0x557eebd5136d <unknown>\n#6 0x557eebd70297 <unknown>\n#7 0x557eebd510e3 <unknown>\n#8 0x557eebd19044 <unknown>\n#9 0x557eebd1a44e <unknown>\n#10 0x557eebfdf861 <unknown>\n#11 0x557eebfe3785 <unknown>\n#12 0x557eebfcd285 <unknown>\n#13 0x557eebfe441f <unknown>\n#14 0x557eebfb120f <unknown>\n#15 0x557eec008028 <unknown>\n#16 0x557eec0081f7 <unknown>\n#17 0x557eec019ed4 <unknown>\n#18 0x7f1672294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"44b46e2e13f5195c","name":"public ip address","source":"44b46e2e13f5195c.txt","type":"text/plain","size":13},{"uid":"e38a85f6d4f63597","name":"Cookies","source":"e38a85f6d4f63597.json","type":"application/json","size":4404},{"uid":"28d99be7d7c2f2ae","name":"Session Storage","source":"28d99be7d7c2f2ae.json","type":"application/json","size":358},{"uid":"e2aa7875ac8afebb","name":"Local Storage","source":"e2aa7875ac8afebb.json","type":"application/json","size":208},{"uid":"584be95c93413e4e","name":"Console Logs","source":"584be95c93413e4e.json","type":"application/json","size":2},{"uid":"5695e8327eab1f69","name":"Full Page Screenshot","source":"5695e8327eab1f69.png","type":"image/png","size":1283197},{"uid":"1d327b598a21e6d3","name":"URL","source":"1d327b598a21e6d3.uri","type":"text/uri-list","size":23},{"uid":"cc3549d5919ef6eb","name":"Network Logs","source":"cc3549d5919ef6eb.json","type":"application/json","size":25925}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Security"},{"name":"tag","value":"Tagged test"},{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"},{"name":"tag","value":"flaky(reruns=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az1153-24"},{"name":"thread","value":"2250-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[{"uid":"65a0dc766fed1a70","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x56528ebc7d33 <unknown>\n#1 0x56528e884f87 <unknown>\n#2 0x56528e8d4c0e <unknown>\n#3 0x56528e8d4d11 <unknown>\n#4 0x56528e920004 <unknown>\n#5 0x56528e8fe36d <unknown>\n#6 0x56528e91d297 <unknown>\n#7 0x56528e8fe0e3 <unknown>\n#8 0x56528e8c6044 <unknown>\n#9 0x56528e8c744e <unknown>\n#10 0x56528eb8c861 <unknown>\n#11 0x56528eb90785 <unknown>\n#12 0x56528eb7a285 <unknown>\n#13 0x56528eb9141f <unknown>\n#14 0x56528eb5e20f <unknown>\n#15 0x56528ebb5028 <unknown>\n#16 0x56528ebb51f7 <unknown>\n#17 0x56528ebc6ed4 <unknown>\n#18 0x7fee4de94ac3 <unknown>","time":{"start":1703639567680,"stop":1703639577883,"duration":10203}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","flaky(reruns=1)"]},"source":"a05a62543e48e6af.json","parameterValues":[]}